.github/workflows/ci.helm-lint.yaml

```diff
@@ -0,0 +1,48 @@
+name: CI Helm Lint
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+      - gcp-cypress-prod
+      - gcp-baobab-prod
```

Comment:
I think we don't want to have this with `gcp-baobab-prod` 🤔

Reply:
I'll remove `main` and `gcp-baobab-prod` in the cypress workflow.